components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -412,6 +432,30 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     }
 
     this._appendLine('Proceeding with project number: ' + projectNumber);
+    const token = await Gapi.getToken();
+    await request(
+      {
+        body: JSON.stringify(
+          {
+            Project: project,
+            ProjectNumber: projectNumber,
+            Token: token,
+          }
+        ),
+        headers: { 'content-type': 'application/json' },
+        method: 'PUT',
+        uri: this._configSpec.appAddress + '/initProject',
+      },
+      (error, response, body) => {
+        if (!error) {
+          const msg = JSON.parse(response.body).Reply;
```

Comment:
Let's always wrap `JSON.parse` calls in a `try/catch`, especially since we don't have string API typings (protos or such). If the server decides to return something else here, the client should handle this gracefully by showing an error dialog, instead of things just stopping to work.